### DISCUSSION TITLE
How should the value look for the 'other' option in a Radio field?

### DISCUSSION THREAD

**Discussion by Gytjarek** [harness-software/wp-graphql-gravity-forms#390] [Q&A]
I'm attempting to send a mutation for a Radio field where the `other` option is selected. Gravity Forms sends`gf_other_choice` along with the `input_{$field_id}_value`. However, I noticed that the Radio input supports only a single value field in the mutation. How can I bypass this limitation?
`mutation {
  submitGfForm(input: {fieldValues: [{value: "gf_other_choice",  "HOW TO PASS OTHER VALUE HERE" id: 7}], id: "1"}) {
    errors {
      message
      id
    }
  }
}`

**Comment by justlevine** [comment]
Hey @Gytjarek , sincerest apologies that I did not see this sooner.

You actually discovered a bug. The syntax is supposed to be `{ id: "1", value: "My custom other value" }`, but that actually broke in a previous GF Core release 🤦

A fix is up in #421 , which will be included in the upcoming v0.13.x release.
